kctf-operator/config/samples/mychal.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 
```

Comment:
should we remove these mychal.yaml files?

Reply:
I used them for testing, and it's the convention I think :D